paasta_tools/cli/cmds/security_check.py

```diff
@@ -15,29 +15,38 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+from paasta_tools.utils import _run
 from paasta_tools.utils import paasta_print
+from paasta_tools.utils import load_system_paasta_config
 
 
 def add_subparser(subparsers):
     list_parser = subparsers.add_parser(
         'security-check',
-        description='Performs a security check (not implemented)',
-        help='Performs a security check (not implemented)',
+        description='Performs a security check (alpha)',
+        help='Performs a security check (alpha)',
     )
     list_parser.add_argument(
         '-s', '--service',
         help='Name of service for which you wish to check. Leading "services-", as included in a '
              'Jenkins job name, will be stripped.',
-        required=True,
+        required=False,
     )
     list_parser.add_argument(
         '-c', '--commit',
         help='Git sha of the image to check',
-        required=True,
+        required=False,
     )
     list_parser.set_defaults(command=perform_security_check)
 
 
 def perform_security_check(args):
-    paasta_print('Not implemented yet')
+    security_check_command = load_system_paasta_config().get_security_check_command()
+    if not security_check_command:
+        paasta_print("Nothing to be executed during the security-check step")
+        return 0
+
+    ret_code, output = _run(security_check_command)
+    paasta_print("{0} exited with {1}: {2}", security_check_command, ret_code, output)
```

Comment:
This is okay like this. The following is just if you want to take a look, I'm happy for it to ship as is :-)

You might want to look at is the args for `_run`:
https://github.com/Yelp/paasta/blob/master/paasta_tools/utils.py#L1024

I'm not super familiar with it but it looks like there's some logic for streaming the output so you don't have to wait for the command to complete to see the results. You could use that and then just print the return code here. It also looks like it has a timeout option. That might be worth enabling because then we could be sure that this step won't run for more than x minutes which would mean we won't hold up pipelines in the case when something gets stuck.